saas/urls/api/subscriber/profile.py

```diff
@@ -45,6 +46,9 @@
         name='saas_api_subscription_list'),
     url(r'^profile/(?P<organization>%s)/?$' % ACCT_REGEX,
         OrganizationDetailAPIView.as_view(), name='saas_api_organization'),
+    url(r'^profile/(?P<organization>%s)/picture$' % ACCT_REGEX,
```

Comment:
In djaodjin-signup, the end-point pattern is `^contacts/(?P<user>%s)/picture/`. It seems natural to add a trailing '/' here.